source/Calamari.Tests/KubernetesFixtures/ManifestReporterTests.cs

```diff
@@ -83,63 +92,31 @@ public void GivenInValidManifest_ShouldNotPostServiceMessage(string enabledFeatu
 
         [TestCase(nameof(FeatureToggle.KubernetesLiveObjectStatusFeatureToggle))]
         [TestCase(OctopusFeatureToggles.KnownSlugs.KubernetesObjectManifestInspection)]
-        public void GivenNamespaceInManifest_ShouldReportManifestNamespace(string enabledFeatureToggle)
```

Comment:
We don't test this kind of logic in the manifest reporter as this is the responsibility of the `KubernetesManifestNamespaceResolver`